TypeCobol/TypeCobol.csproj

```diff
@@ -66,7 +66,397 @@
     <Compile Include="$(TC_AntlrGeneratedFiles)\TypeCobolCodeElementsVisitor.cs">
       <Link>Compiler\Parser\Generated\TypeCobolCodeElementsVisitor.cs</Link>
     </Compile>
-    <Compile Include="**\*.cs" />
+    <Compile Include="Analysis\AnalyzerProviderWrapper.cs" />
```

Comment:
Please revert changes. csproj files are not meant to be modified, the cs files are automatically included through wildcards